src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Assembly.cs

```diff
@@ -370,7 +370,7 @@ void M(de x)
 
             compilation = CreateStandardCompilation("", new MetadataReference[] { compilation.EmitToImageReference() }, TestOptions.ReleaseDll, assemblyName: assemblyNameBase + "20");
 
-            CompileAndVerify(compilation, verify: false).VerifyDiagnostics(
+            CompileAndVerify(compilation, verify: Verification.Skipped).VerifyDiagnostics(
```

Comment:
:memo: I didn't spend much time investigating places that caused trouble (desktop vs Core: fail on one platform but pass on the other), just marked them as Skipped, which maintains the old behavior. There's about 20 of them (out of 400).